src/application-base/ApplicationBase.jsx

```diff
@@ -119,12 +119,11 @@ const ApplicationBase = ({
 
   return (
     <div data-terra-application-base className={cx('application-base', { fill: !fitToParentIsDisabled })}>
-      <ThemeProvider
-        themeName={themeName}
-      >
+      <ThemeProvider themeName={themeName}>
         <ThemeContextProvider theme={theme}>
           <Base
             customMessages={customTranslatedMessages}
+            throwOnI18nLoadError
```

Comment:
should this be caught by application error boundary ? i know currently base is not wrapped by error boundary. but with introduction of this do we need to wrap base also ?

Reply:
I originally had the same thought. The application error boundary relies on translations so we would have to introduce another error boundary for this level. I talked offline with @tbiethman and @mjhenkes  about this and we felt if a failure occurs at this level, there isn't much we can do to save the application since locales/translations data is needed for the app to render and function as expected.